### PR TITLE
Question selection (wip)

### PR DESCRIPTION
**Pull request by ngodfraind**
- needs to be tested
- documentation ?

### CODE REVIEW

File: format/answer-data/selection/schema.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          }
+        },
+        {
+          "$schema": "http://json-schema.org/draft-04/schema#",
+          "type": "object",
+          "properties": {
+            "data": {
+              "type": "array",
+              "items": {
+                 "type": "integer"
```
Comment (stefk) [inline format/answer-data/selection/schema.json]: Always use strings for ids.

File: format/question/selection/examples/valid/invisible.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "id": "1",
+  "type": "application/x.selection+json",
+  "content": "Question ?",
+  "text": "Lorem ipsum sit amet.",
+  "mode": "find",
+  "tries": 10,
+  "solutions":  [
+    {
+      "selectionId": "select1",
+      "answers": [
```
Comment (stefk) [inline format/question/selection/examples/valid/invisible.json]: I don't think an array of answers makes sense here. You should inline `score`, `begin` and `end` in the parent object.

File: format/question/selection/schema.json [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+                    "minItems": 1,
+                    "uniqueItems": true,
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "colorId": {
+                          "type": "string"
+                        },
+                        "score": {
+                          "type": "integer"
```
Comment (stefk) [inline format/question/selection/schema.json]: Too restrictive, use `number` instead.

File: format/question/selection/schema.json [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+                      "type": "object",
+                      "properties": {
+                        "begin": {
+                          "type": "integer"
+                        },
+                        "end": {
+                          "type": "integer"
+                        },
+                        "score": {
+                          "type": "integer"
```
Comment (stefk) [inline format/question/selection/schema.json]: Same.

File: format/question/selection/schema.json [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+              "minItems": 1,
+              "uniqueItems": true,
+              "items": {
+                "type": "object",
+                "properties": {
+                  "selectionId": {
+                    "type": "string"
+                  },
+                  "score": {
+                    "type": "integer"
```
Comment (stefk) [inline format/question/selection/schema.json]: Same.

File: format/answer-data/selection/schema.json (hunk 1 above)
Comment (ngodfraind) [inline format/answer-data/selection/schema.json]: I wanted to save the positons where the user clicked (for the feedback/paper).

Comment (stefk) [inline format/answer-data/selection/schema.json]: Right, I misunderstood that part.

**Pull request merged**